api/src/db/utils.ts

```diff
@@ -43,3 +43,9 @@ export const generateNamespacePrefix = (len: number = 6): string => {
     .toString('hex')
     .slice(0, len);
 };
+
+export const comparerContact = (otherArray) =>{
```

Comment:
This could use a bit of a clean up, it is a little tough to understand what is going on with it.
title, spacing, missing type, missing ;, variables are unclear (if the method is called compareContacts then the internal variables should reflect this), 
```suggestion
export const compareContacts = (otherArray) => {
```

Reply:
updated to use lodash instead of self-writing function